<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Node_Dialog</class>
 <widget class="QDialog" name="Edit_Node_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>499</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="node_box">
       <property name="title">
        <string>Node</string>
       </property>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>339</width>
          <height>149</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="name_label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="weight_label">
           <property name="text">
            <string>Weight</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="weight_box">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="domain_label">
           <property name="text">
            <string>Domain</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="domain_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="name_line"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="resource_box">
       <property name="title">
        <string>Resources</string>
       </property>
       <widget class="QTableView" name="resources_table">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>341</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="add_resource_button">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>390</y>
          <width>114</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_resource_button">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>390</y>
          <width>114</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="control_buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>control_buttons</sender>
   <signal>accepted()</signal>
   <receiver>Edit_Node_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>control_buttons</sender>
   <signal>rejected()</signal>
   <receiver>Edit_Node_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
